src/librustc_driver/Cargo.toml

```diff
@@ -35,3 +35,4 @@ serialize = { path = "../libserialize" }
 syntax = { path = "../libsyntax" }
 syntax_ext = { path = "../libsyntax_ext" }
 syntax_pos = { path = "../libsyntax_pos" }
+isatty = "0.1"
```

Comment:
@Mark-Simulacrum I'm not 100% sure about the version here. Should I make it strict?